packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -503,6 +503,8 @@ class _AndroidViewState extends State<AndroidView> {
   @override
   void dispose() {
     _controller.dispose();
+    _focusNode?.dispose();
+    _focusNode = null;
```

Comment:
@goderbauer I'm sure it's harmless but is it idiomatic in the framework to set the property to null after disposal? That seems unnecessary but I see the both patterns sprinkled around.

@Lionyy this is in `_AndroidViewState`, did you just audit all `focusNode` usages, or were you actually seeing a leak?  It would be helpful in the description to describe what exactly this PR is doing.

Reply:
It's fairly idiomatic to ensure that nobody uses an object accidentally after dispose.

Reply:
Very interesting! In objc's `dealloc`, there is no way to access an ivar after the object itself is deallocated, so we actually don't have to do `_ivar = nil` in `dealloc`. 

Though in objc `dealloc` is called automatically when the object is gone, but here we have to manually call `dispose` in dart. So they are different.